sdk/transaction_model.go

```diff
@@ -3282,6 +3302,12 @@ func MapTransaction(b *bytes.Buffer, generationHash *Hash) (Transaction, error)
 		dto = &placeSdaOfferTransactionDTO{}
 	case RemoveSdaExchangeOffer:
 		dto = &removeSdaExchangeOfferTransactionDTO{}
+	case AutomaticExecutionsPayment:
+		dto = &automaticExecutionsPaymentTransactionDTO{}
+	case ManualCall:
+		dto = &manualCallTransactionDTO{}
+	case DeployContract:
+		dto = &deployContractTransactionDTO{}
```

Comment:
also, add `Unsuccessful and SuccessfulEndBatchExecution`

Reply:
https://github.com/proximax-storage/go-xpx-chain-sdk/commit/622604afaa00afc762c77f69247af97dffa8b036